aws/aws-parallelcluster
# [Isolated Regions] Add support for isolated regions: us-iso-* and us-isob-*

### Description of changes
Added support for isolated regions: us-iso-* and us-isob-*

### Tests
* Unit tests
* Manual test creating a cluster
* Full regression tests will be done by the authoritative pipeline

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Signed-off-by: Giacomo Marciani <mgiacomo@amazon.com>
